src/ShopifyApp/resources/views/partials/token_handler.blade.php

```diff
@@ -23,7 +23,13 @@
 
         if (window.jQuery) {
             // jQuery
-            window.jQuery.ajaxSettings.headers['Authorization'] = `Bearer ${window.sessionToken}`;
+            if (window.jQuery.ajaxSettings.headers) {
+                window.jQuery.ajaxSettings.headers['Authorization'] = `Bearer ${window.sessionToken}`;
+            } else {
+                window.jQuery.ajaxSetup({
+                    headers: { 'Authorization': 'Bearer ' + window.sessionToken}
```

Comment:
@thang12l Can you adjust this line to use template syntax like above :) I can merge then.